src/tile/tile.ts

```diff
@@ -0,0 +1,32 @@
+import chalk from "chalk";
+
+enum Colors {
+    'yellow',
+    'blue',
+    'red',
+    'green',
+    'cyan',
+    'white'
+}
+
+enum Symbols {
+    A = 'A',
+    B = 'B', 
+    C = 'C', 
+    D = 'D', 
+    E = 'E',
+    F = 'F'
+}
+
+export interface tileDataInterface {
+    color: number,
+    symbol: string
+}
+
+/**
+ * @description Draw text from a tile object.
+ */
+export const drawChalkText = (tileData: tileDataInterface): string => {
+    const drawTextFunction = chalk[Colors[tileData.color]];
```

Comment:
If you use an enum for `Color` and set the `tileData.color` type to `Color`, this code can be simplfied to `chalk[tileData.color]` which is more readable imho

Reply:
Done